src/Database/Commands/PatchByIndexCommand.ts

```diff
@@ -29,17 +29,16 @@ export class PatchByIndexCommand extends IndexQueryBasedCommand {
   }
 
   public setResponse(response: IResponse): IRavenResponse | IRavenResponse[] | void {
-    const responseBody: IResponseBody = response.body;
-    const status: StatusCode = response.statusCode;
+    const result: IRavenResponse = <IRavenResponse>super.setResponse(response);
 
-    if(!responseBody) {
-      throw new ErrorResponseException(StringUtil.format('Could not find index {0}', this.indexName));
+    if(!response.body) {
+      throw new IndexDoesNotExistException(StringUtil.format('Could not find index {0}', this.indexName));
     }
 
-    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(status)) {
+    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(response.statusCode)) {
```

Comment:
OK I know what's going on. When you target es2016 it's fine. That's because `.includes()` is a part of ES7 spec, not ES6. Not sure if Node 6 is ES2016 though?